### PR TITLE
merge myoclonic epilepsy of Lafora 2

### LINKED ISSUE #7393: MYOCLONIC EPILEPSY OF LAFORA 2, MONDO:0958201 & MONDO:0800306 [Merge]
**Mondo term (ID and Label)**
MONDO:0958201 myoclonic epilepsy of Lafora 2

**Reason for deprecation**
This is a new OMIM record for the previous 'included' concept: https://www.omim.org/entry/620681 
# 620681- MYOCLONIC EPILEPSY OF LAFORA 2; MELF2

Alternative titles; symbols
EPILEPSY, PROGRESSIVE MYOCLONIC, 2B; EPM2B
LAFORA DISEASE 2

**Term to be merged with**
MONDO:0800306 epilepsy, progressive myoclonic, 2b

**Your nano-attribution (ORCID)**
If you don't have an ORCID, you can sign up for one [here](https://orcid.org/)

### PR DESCRIPTION
**Pull request by nicolevasilevsky**
close #7393

### CODE REVIEW

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -526302,14 +526302,21 @@ property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/
...
 id: MONDO:0800306
-name: epilepsy, progressive myoclonic, 2b
+name: myoclonic epilepsy of Lafora 2
+def: "Any Lafora disease in which the cause of the disease is a variation in the NHLRC1 gene." [MONDO:patterns/disease_series_by_gene]
 subset: gard_rare {source="MONDO:GARD"}
 subset: inferred_rare
 subset: rare
+synonym: "epilepsy, progressive myoclonic, 2B" EXACT [OMIM:620681]
 synonym: "EPM2B" EXACT ABBREVIATION [OMIM:254780]
-xref: OMIM:254780 {source="MONDO:includedEntryInOMIM"}
```
Comment (twhetzel) [inline src/ontology/mondo-edit.obo:526310]: Why was this removed? Is this [subset](https://mondo.readthedocs.io/en/stable/editors-guide/f-entities/#subsets) still being used in genera?

File: src/ontology/mondo-edit.obo [hunk 2]
```diff
@@ -526302,14 +526302,21 @@ property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/
 
 [Term]
 id: MONDO:0800306
-name: epilepsy, progressive myoclonic, 2b
+name: myoclonic epilepsy of Lafora 2
+def: "Any Lafora disease in which the cause of the disease is a variation in the NHLRC1 gene." [MONDO:patterns/disease_series_by_gene]
```
Comment (twhetzel) [inline src/ontology/mondo-edit.obo:526306]: Should this term also have the `has material basis in germline mutation in` property?

File: src/ontology/mondo-edit.obo [hunk 3]
```diff
@@ -535301,6 +535308,7 @@ is_a: MONDO:0030309 {source="OMIM:620569"} ! Leber hereditary optic neuropathy,
 [Term]
 id: MONDO:0958199
 name: myoclonic epilepsy of Lafora 1
+def: "Any Lafora disease in which the cause of the disease is a variation in the EPM2A gene." [MONDO:patterns/disease_series_by_gene]
```
Comment (twhetzel) [inline src/ontology/mondo-edit.obo:535311]: Should this term also have the `has material basis in germline mutation in` property?

**Review by twhetzel — APPROVED** [review]
Approving, but a few questions that may need an update to address.

File: src/ontology/mondo-edit.obo (hunk 1 above)
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:526310]: yes, it is still being used but it is no longer the case that this OMIM is an included entry in OMIM on this class

File: src/ontology/mondo-edit.obo (hunk 2 above)
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:526306]: added

File: src/ontology/mondo-edit.obo (hunk 3 above)
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:535311]: added
